api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -375,6 +384,9 @@ public void setUserProvider(Provider<User> userProvider) {
     cohortReviewService.deleteParticipantCohortAnnotation(
         annotationId, cohortReviewId, participantId);
 
+    Workspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceId);
+    workspaceService.updateRecentWorkspaces(workspace.getWorkspaceId(), userProvider.get().getUserId());
```

Comment:
Should we maybe be updating this new table and the main workspace table in a single transaction?